### DISCUSSION TITLE
Allow users to attach SVG images in the Markdown editor

### DISCUSSION THREAD

**Discussion by Arnie97** [community/community#4816] [Other Feature Feedback, Questions, & Ideas]
I was trying to upload a SVG vector image (generated by [pprof](https://github.com/google/pprof)) in discussions, but I was told that only GIF, JPEG, PNG and some other file types were supported. Since SVGs are technically XML text files, they tend to be much smaller than the bitmap alternatives. Were there any concerns to exclude SVG from the file type list?

**Comment by mririgoyen** [comment]
As a core contributor to [Material Design Icons](https://github.com/Templarian/MaterialDesign/issues), we would much appreciate the ability for us to allow our potential contributors the ability to directly attach SVG files to issues without the need to ZIP them up first.

Our community is driven by issues and a core team is responsible for vetting and adding icons to the library through a custom built CMS. This happens by a special [Contribution Issue Template](https://github.com/Templarian/MaterialDesign/issues/new?assignees=&labels=Icon+Request%2CContribution&template=2_contribution.yml) that instructs the contributor to attach their SVG to the issue for consideration. Requiring the compression of the source file to upload it at this point is another barrier to contributing we would like to remove.

Please consider adding support for SVGs, even if they act simply as attachments as opposed to displaying their image contents.
